testdata/networking/networkpolicy/allow-ns-and-pod.yaml

```diff
@@ -0,0 +1,19 @@
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: allow-ns-and-pod
+spec:
+  podSelector:
+    matchLabels:
+      name: test-pods
+  ingress:
+    - from:
+      - namespaceSelector:
+          matchLabels:
+            team: operations
+        podSelector:
+          matchLabels:
+            name: test-pods
+  podSelector:
```

Comment:
line '17-19' should be dup with line '6-8'

Reply:
Fixed